src/pconn.cc

```diff
@@ -348,33 +350,30 @@ PconnPool::key(const Comm::ConnectionPointer &destLink, const char *domain)
 }
 
 void
-PconnPool::dumpHist(StoreEntry * e) const
+PconnPool::dumpHist(std::ostream &yaml) const
 {
-    storeAppendPrintf(e,
-                      "%s persistent connection counts:\n"
-                      "\n"
-                      "\t Requests\t Connection Count\n"
-                      "\t --------\t ----------------\n",
-                      descr);
+    AtMostOnce title("  persistent connection counts:\n");
+    IfNoOutput defaultText("  {}\n", yaml);
 
     for (int i = 0; i < PCONN_HIST_SZ; ++i) {
         if (hist[i] == 0)
             continue;
 
-        storeAppendPrintf(e, "\t%d\t%d\n", i, hist[i]);
+        yaml << title << defaultText <<
+             "    " << i << ": " << hist[i] << "\n";
     }
 }
 
 void
-PconnPool::dumpHash(StoreEntry *e) const
+PconnPool::dumpHash(std::ostream &yaml) const
 {
-    hash_table *hid = table;
+    auto *hid = table;
     hash_first(hid);
+    AtMostOnce title("  pool hash table:\n");
```

Comment:
```yaml
  pool hash table:
    - "[::1]:3129"
    - "172.217.25.196:80/www.google.com"
```

I do not see a table. I see a list (of addresses).

Also, "hash" is an internal implementation detail that may change and that readers should not care about. Please remove that word as noise.